spikeextractors/extractors/neuroscopeextractors/neuroscopeextractors.py

```diff
@@ -397,15 +403,21 @@ def __init__(self, folder_path: PathType, keep_mua_units: bool = True, exclude_s
             'samplingRate').text)  # careful not to confuse it with the lfpsamplingrate
 
         res_files = [f for f in folder_path.iterdir() if f.is_file()
-                     and '.res' in f.suffixes and '.temp' not in f.suffixes and len(f.suffixes) <= 2]
+                     and '.res' in f.suffixes 
+                     and '.temp' not in f.suffixes
+                     and not f.name.endswith('~')
```

Comment:
What would you think about adding `re` as an import, and replacing these two lines with

` re.search(r'\d+$', f.name) is not None`

Basically, it just makes these checks a hard enforcement of the .res.%i format. If it ends in ~ or anything else (which has been observed before), it won't pass the check.